static/sass/_pattern_subscribe.scss

```diff
@@ -2,6 +2,41 @@
   // The default card padding.
   $card-padding: calc(#{$spv-inner--large} - 1px);
 
+  .p-subscriptions__header {
+    // Put the title and buttons on the same line.
+    display: flex;
+    // Space the buttons menu to the right of the container.
+    justify-content: space-between;
+
+    @media only screen and (max-width: $breakpoint-medium) {
+      // At medium screens return the title and buttons to be stacked above
+      // each other.
+      display: unset;
+    }
+  }
+
+  .p-subscriptions__header-buttons {
+    // Move the buttons down to match the title.
+    padding-top: $spv-outer--scaleable;
+
+    @media only screen and (max-width: $breakpoint-small) {
+      // At small screens make the buttons closer to the title.
+      padding-top: $spv-outer--medium;
+
+      [class^="p-button"] {
+        // At small screens make the buttons fill the screen.
+        width: 100%;
+      }
+    }
```

Comment:
This already happens for `$breakpoint-x-small` is that a requirement that they should do that for `small` as well?

Reply:
Yeah OK, I'll remove that and try and get a design review at some point for the in-between screen sizes.